payroll/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Payroll",
-    "version": "15.0.1.1.1",
+    "version": "15.0.1.2.0",
```

Comment:
Please do not update the version. The bot will do it automatically.

Reply:
The manual bump is correct, as there are migration scripts joined to that version, and launching incorrectly the bot command will make them useless. Please remove your review and merge with `nobump`.